packages/page-accounts/src/Sidebar/AddressSection.tsx

```diff
@@ -1,12 +1,17 @@
 // Copyright 2017-2022 @polkadot/page-accounts authors & contributors
 // SPDX-License-Identifier: Apache-2.0
 
-import React from 'react';
+//import React from 'react';
+import React, { useState } from 'react';
+
+import { Button } from '@polkadot/react-components';
 
 import { AccountName, IdentityIcon, Input } from '@polkadot/react-components';
 import { AddressFlags } from '@polkadot/react-hooks/types';
 
 import { useTranslation } from '../translate';
+import {CopyToClipboard} from 'react-copy-to-clipboard';
```

Comment:
```suggestion
import { CopyToClipboard } from 'react-copy-to-clipboard';
```
Additionally it would need to move to just below the `react` usage above.